src/components/CommentAdder.jsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react"
+import { postNewComment } from "../utils/utils"
+import Comments from "./Comments"
+
+
+const CommentAdder = ({review_id}) => {
+
+    const [newComment, setNewComment] = useState( {
+        username:'',
+        body: ''
+        })
+
+    const [isPosted, setIsPosted]=useState(true)
+    const [err, setErr] = useState(null)  
+
+
+    function handleCommentChange(event){
+        setNewComment((currComment) => {return {...currComment, [event.target.id]:event.target.value}})
+    }
+
+ 
+    function handleSubmit(event){
+        event.preventDefault()
+        setErr(null)
+   
+        setIsPosted(false)
+       
+        postNewComment(newComment, review_id).then((comment) => {
+            
+            setIsPosted(true);
+            setNewComment((currComment) => {return {username:'', body:''}})
+          }).catch(err => {
+            setIsPosted('not')
```

Comment:
You're using a boolean value above and then a string to handle whether something is posted or not, this is quite confusing. Choose one method or another